doc/en/reference/customize.rst

```diff
@@ -177,6 +177,9 @@ Files will only be matched for configuration if:
 * ``tox.ini``: contains a ``[pytest]`` section.
 * ``setup.cfg``: contains a ``[tool:pytest]`` section.
 
+Finally, a ``pyproject.toml`` file will be considered the ``configfile`` if no other match was found, in this case
+even if it does not contain a ``[tool.pytest.ini_options]`` table (this was added in ``8.1``).
```

Comment:
```suggestion
Finally, a ``pyproject.toml`` file will be considered the ``configfile`` if no other match was found, 
even if it does not contain a ``[tool.pytest.ini_options]`` table (this was added in ``8.1``).  It will thus define the root dir.
```
It was unclear for me at first what the purpose of a pyproject.toml without any pytest related config inside was.

Reply:
Thanks! Used a slightly different approach, moving this to its own line as it is independent from `pyproject.toml` itself.